{"name": "test_register_with_duplicate_fields", "status": "broken", "statusDetails": {"message": "AttributeError: 'SubRequest' object has no attribute 'mode'. Did you mean: 'node'?", "trace": "self = <allure_commons._allure.fixture object at 0x0000022CD482C9D0>, args = (), kwargs = {}\n\n    def __call__(self, *args, **kwargs):\n        self.parameters = func_parameters(self._fixture_function, *args, **kwargs)\n    \n        with self:\n>           return self._fixture_function(*args, **kwargs)\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n.venv\\Lib\\site-packages\\allure_commons\\_allure.py:231: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nrequest = <SubRequest 'log_on_failure' for <Function test_register_with_duplicate_fields>>\n\n    @pytest.fixture()\n    def log_on_failure(request):\n        yield\n>       item=request.mode\n             ^^^^^^^^^^^^\nE       AttributeError: 'SubRequest' object has no attribute 'mode'. Did you mean: 'node'?\n\ntests\\conftest.py:11: AttributeError"}, "start": 1758948703854, "stop": 1758948707662, "uuid": "f64f9cfe-15d7-481e-ae1c-068f780cba5b", "historyId": "c2d6bff9c2d424ef21711c4ad199a821", "testCaseId": "c2d6bff9c2d424ef21711c4ad199a821", "fullName": "tests.test_register.TestRegister#test_register_with_duplicate_fields", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_register"}, {"name": "subSuite", "value": "TestRegister"}, {"name": "host", "value": "DESKTOP-PQJ2IUR"}, {"name": "thread", "value": "12224-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_register"}], "titlePath": ["tests", "test_register.py", "TestRegister"]}